### PR TITLE
EVG-7660: add backup cli tool

### PR DESCRIPTION
**Pull request by tychoish**

### CODE REVIEW

File: operations/admin.go [hunk 1]
```diff
@@ -476,3 +481,70 @@ func updateRoleCmd() cli.Command {
...
+		Usage: "create a backup (to s3) of a collection",
+		Flags: mergeFlagSlices(
+			serviceConfigFlags(),
+			addDbSettingsFlags(
+				cli.StringFlag{
+					Name:  joinFlagNames(collectionNameFlag, "-c"),
+					Usage: "specify the name of the collection to backup",
+				},
+				cli.StringFlag{
+					Name:  "prefix",
```
Comment (bsamek) [inline operations/admin.go]: `prefixFlagName`

File: operations/admin.go [hunk 2]
```diff
@@ -476,3 +481,70 @@ func updateRoleCmd() cli.Command {
...
+			),
+		),
+		Before: requireStringFlag(collectionNameFlag),
+		Action: func(c *cli.Context) error {
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			collectionName := c.String(collectionNameFlag)
+
+			env, err := evergreen.NewEnvironment(ctx, c.String(confFlagName), parseDB(c))
```
Comment (bsamek) [inline operations/admin.go]: Isn't this the wrong conf to construct an environment from? This is the user's local conf. Is your plan to provide one on an app server for the db values?

Comment (tychoish) [reply operations/admin.go]: nope. that's c.Parent().String(confFlagName)

Comment (tychoish) [reply operations/admin.go]: this one comes from the serviceConfigFlags

**Pull request merged**